benchmark/src/index.ts

```diff
@@ -0,0 +1,77 @@
+import envinfo from 'envinfo';
+import fetch from 'node-fetch';
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
+import { exit } from 'process';
+import { benchmark } from './benchmark';
+import Renderer from './renderer';
+
+const svgUrlMap = {
+  icon: 'https://github.com/ssssota/svg2png-wasm/raw/main/logo.svg',
+  tiger: 'https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/tiger.svg',
+};
```

Comment:
Please consider using local files to avoid different network speeds at different times affecting the test results.